libs/datamodel/core/tests/reformat/reformat.rs

```diff
@@ -855,13 +847,6 @@ fn multiple_new_lines_between_top_level_elements_must_be_reduced_to_a_single_one
         // free floating comment
         /// free floating doc comment
 
-        // type alias comment
-        /// type alias doc comment
-        type                       MyString = String @default("FooBar")
-
-        // free floating comment
-        /// free floating doc comment
```

Comment:
(the second one)

Reply:
Same here. We don't need these because we still have free floating comments before every root level element.